src/query/plan/variable_start_planner.cpp

```diff
@@ -155,6 +155,18 @@ auto ExpansionNodes(const std::vector<Expansion> &expansions, const SymbolTable
   return nodes;
 }
 
+FilterMatching ToFilterMatching(Matching &matching) {
+  FilterMatching filter_matching;
+  filter_matching.expansions = matching.expansions;
+  filter_matching.edge_symbols = matching.edge_symbols;
+  filter_matching.filters = matching.filters;
+  filter_matching.node_symbol_to_expansions = matching.node_symbol_to_expansions;
+  filter_matching.named_paths = matching.named_paths;
+  filter_matching.expansion_symbols = matching.expansion_symbols;
+
```

Comment:
Do you need to set up here `type`

Reply:
I saw now that this is done in code before

Reply:
but it can not be set because Matching is a superclass. it does not have these types. It is done after

Reply:
yes, I was wrong here